spec/views/idv/inherited_proofing/get_started.html.erb_spec.rb

```diff
@@ -49,5 +49,33 @@
         expect(rendered).to have_content('Empiece con la verificación de su identidad')
       end
     end
+
+    context 'without service provider' do
+      it 'renders troubleshooting options' do
+        render template: 'idv/inherited_proofing/get_started'
+
+        expect(rendered).to have_link(t('inherited_proofing.troubleshooting.options.get_va_help'))
+        expect(rendered).to have_link(
```

Comment:
I thought you were going to fix/change/remove these expectations (cuz they were incorrect).

Reply:
removed from agreement but not here, my bad. done now